### ISSUE TITLE
Missing Aria info for screen readers on Switch

### ISSUE THREAD

**Issue by CalvinConnell** [telerik/kendo-angular#1690]
## I'm submitting a...

* Bug report 

## Current behavior
Screen readers don't receive enough information from aria tags or semantics to operate effectively. 
The name is set to "On/Off" and there is no state or status available to aria attributes

## Expected behavior

Ideally, the functionality would align with buttons and checkboxes, where the state is given as an 
aria attribute, and is updated on change.

## Minimal reproduction of the problem with instructions
https://plnkr.co/edit/42FpGFbtIlcKvkZC9i2V?p=preview
1. Right click on the switch.
2. Click Inspect.
3. Go to the Accessibility tab (if included) and check aria attributes
Or 
1. Get ChromeVox extension and operate the switch with it turned on.
Or 
1. Install NVDA Screen Reader and operate. 

## Environment

Package versions:

```

```

Browser:

- Chrome (desktop) version 67.0.3396.99

**Comment by danielkaradachki** [comment]
Merging from https://github.com/telerik/kendo-angular/issues/2316 - associated labels are not read by the screen readers:

https://stackblitz.com/edit/angular-xfc5fv?file=app/app.component.ts

**Comment by siyka-marinova** [comment]
Fix for this issue is available in the [latest dev. version](https://www.telerik.com/kendo-angular-ui/components/installation/development-builds/#toc-obtaining-development-builds) of `kendo-angular-inputs`

**Comment by tsvetomir** [comment]
Released in v6.5.1
